src/frontend/src/pages/WorkPackageForm/WorkPackageFormView.tsx

```diff
@@ -119,6 +124,76 @@ const WorkPackageFormView: React.FC<WorkPackageFormViewProps> = ({
   } = useFieldArray({ control, name: 'deliverables' });
 
   const { userId } = user;
+  const {
+    data: workPackageTemplates,
+    isLoading: workPackageTemplateisLoading,
+    isError: workPackageTemplateisError,
+    error: workPackageTemplateError
+  } = useAllWorkPackageTemplates();
+  if (workPackageTemplateisLoading || !workPackageTemplates) return <LoadingIndicator />;
+  if (workPackageTemplateisError) return <ErrorPage message={workPackageTemplateError.message} />;
+
+  const testWorkPackageTemplate: WorkPackageTemplate[] = [
```

Comment:
why is there test data in the actual form?

Reply:
There was no seed data for WorkPackageTemplate, and there was no create WorkPackageTemplate endpoint so I couldn't create the seed data and wasn't sure if there was another way to create seed data. So for testing purposes, I just put the test data there. So, once that's created, I can create seed data.

Reply:
hrmm. I think @bderbs30 is working on the create and edit endpoints

Reply:
I'm pretty sure I'm done I'll request review